lib/letsencryptStore.js

```diff
@@ -43,7 +43,7 @@ function writeRenewalConfig(args) {
         , email: args.email
         , tos: args.agreeTos && true
         // yes, it's an array. weird, right?
-        , webrootPath: args.webrootPath && [args.webrootPath] || []
+        , webrootPath: args.webrootPath != undefined ? [args.webrootPath] : []

```

Comment:
The old way to write was pretty weird and one has to think about precedence of `&&` and `||`

Reply:
better readable

Reply:
Please use !==